sig-network/README.md

```diff
@@ -134,6 +134,25 @@ The following [subprojects][subproject-definition] are owned by sig-network:
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 [working-group-definition]: https://github.com/kubernetes/community/blob/master/governance.md#working-groups
 <!-- BEGIN CUSTOM CONTENT -->
+
+### Responsibilities of Subproject Leads
+
+Just like the [SIG Network Chair's](https://github.com/kubernetes/community/blob/master/sig-network/role-sig-chair.md#whats-expected) expectation is to "run the show" of the whole SIG, the Subproject Lead's expectation is to "run the show" for that subproject. This includes:
+
+- **Transparent Project Planning and Maintainence:**
+  - Leads must provide transparent view into their historical and future project plans (eg; using GitHub project boards)
+  - Leads must communicate the project plans to the wider community by linking it to the project's README pages.
+  - Project must be kept in good health by the lead through regular issue triaging and PR reviews (delegating this to other members in the community is good!)
```

Comment:
Check testgrids and jobs failures, CI Health

Reply:
also good point, but I see all these points got overwritten for the final version; I can add this back in @shaneutt are you ok with adding this and the api-review bits that antonio suggested into the list?

Reply:
Sure np